client/service/src/client/wasm_substitutes.rs

```diff
@@ -55,6 +55,7 @@ impl<Block: BlockT> WasmSubstitute<Block> {
 			code_fetcher: self,
 			hash: self.hash.clone(),
 			heap_pages,
+			context: sp_core::traits::CodeContext::Consensus,
```

Comment:
@bkchr a bit uncertain about this too.

Reply:
You need to pass the context here.

Reply:
not sure where that context should be decided from. Can just make it an an argument of the function and propagate it back.

Reply:
done.